src/schema.js

```diff
@@ -52,8 +52,6 @@ const signinSchema = Joi.object({
 
 const createArticleSchema = Joi.object({
 	body:{
-		userId: Joi.number()
```

Comment:
@itzgeebee, this intentional?

Reply:
@jocrah Nah, I probably forgot to add it back after testing something